packages/build/src/build/writePluginImports/writeIconImports.js

```diff
@@ -17,17 +17,14 @@
 import { nunjucksFunction } from '@lowdefy/nunjucks';
 
 const template = `{%- for package in packages -%}
-{%- for icon in package.icons -%}
-import { {{ icon }} } from '{{ package.package }}';
-{% endfor -%}
-{% endfor -%}
+{% if package.icons.length %}import { {% endif %}{%- for icon in package.icons -%}{% if not loop.last -%} {{ icon }}, {% else -%} {{ icon }} } from '{{ package.package }}';
```

Comment:
Please move `endif`  for the `if package.icons.length` to the end of the block as that will be easier to understand.